CIPs/cip-split_etherbase.md

```diff
@@ -0,0 +1,48 @@
+---
+cip: <to be assigned>
+title: Split etherbase
+author: Andres di Giovanni (@andres-dg)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category: Ring 0
+created: 2020-11-19
+license: Apache 2.0
+---
+
+## Simple Summary
+
+Validators maintain the network and are, of course, rewarded for it. If elected during an epoch, they receive validator rewards by the end of it. Additionally, if they are chosen to create the next block, they receive transaction fees. These rewards go to a single configured address in the node, the etherbase. This proposal suggests splitting the etherbase into separate addresses, one for each function the node carries out in the network.
+
+## Abstract
+
+During an epoch, a validator is elected to sign every consensus message. This work is rewarded at the end of the epoch, through validator rewards. The validator is also chosen to process incoming transactions by mining a block. The block creator receives transaction fees for the work done. Currently, geth allows the user to configure a single address, etherbase, to receive both validator rewards and mining transaction fees.
+
+## Motivation
+
+Different rewards get transferred to the same address and the user may want to split them. 
+
+## Specification
+
+Add a flag for each required address:
+    - tx-fee-recipient: the node's coinbase while creating blocks
+    - miner.validator: validator address for epoch rewards
```

Comment:
what's the behavior after donut activates if the `tx-fee-recipient` address is not specified?